deepchem/data/data_loader.py

```diff
@@ -437,6 +439,69 @@ def _featurize_shard(self, shard):
     return (X, np.ones(len(X), dtype=bool))
 
 
+class JsonLoader(DataLoader):
```

Comment:
Could you add a unit test for `JsonLoader` that applies it to some representative materials data? Like on a tiny example

Reply:
Will do!